config/config.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	patJobName    = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9_-]*$`)
+	patJobName    = regexp.MustCompile(`^[a-zA-Z0-9_][a-zA-Z0-9_-]*$`)
```

Comment:
This represents a history of changes. You can see the state change to "Approved" below.